### PR TITLE
[JENKINS-32211] Add `tokenize` to the whitelist.

### PR DESCRIPTION
**Pull request by andresrc**
[JENKINS-32211](https://issues.jenkins-ci.org/browse/JENKINS-32211)

Add `tokenize` to the whitelist.

@reviewbybees

### CODE REVIEW

**Comment by andresrc** [comment]
@reviewbybees

**Comment by oleg-nenashev** [comment]
:bee: . BTW whitelisting of Java system classes is a never-ending story

**Comment by andresrc** [comment]
Yes, it seems to be first come, first served :)

File: src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java [hunk 1]
```diff
@@ -499,6 +499,13 @@ public Object invokeMethod(String name, Object args) {
         return x + 2;
     }
 
+    @Issue("JENKINS-32211")
+    @Test public void tokenize() throws Exception {
+        assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize().size()");
+        assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize(' ').size()");
+        assertEvaluate(new GenericWhitelist(), 3, "'foo bar baz'.tokenize('ba').size()");
+    }
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java]: Do not really need test coverage for simple whitelist additions---only in cases where we think something funny is going on in the code.

**Comment by jglick** [comment]
:bee:

**Pull request merged**